src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+const SEX_MALE = 'm';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === SEX_MALE ? 'wife' : 'husband'}`}
+        </p>
+      ) : (
+        <p className="Person__partner">
+          I am not married
```

Comment:
In both cases you have a Person__partner paragraph. You can move ternary operator inside it to make your code a bit cleaner